src/styles/variables.sass

```diff
@@ -27,9 +27,9 @@ $c-main: green
 $c-comp: #333
 $c-alt: #000
 
-$text-color: $c-comp
-$text-color-light: #676767
-$text-back: #FFF
+$c-text: $c-comp
+$c-text-light: #676767
+$c-text-back: #FFF
```

Comment:
Já bych ideálně z `$c-text-back` vyhodil to `-text`. Nedává mi tam příliš smysl nebo používám tuhle proměnnou špatně, ale vnímám ji obecněji než jen jako pozadí pro text.